src/services/notification/discord.rs

```diff
@@ -210,7 +199,7 @@ mod tests {
 		variables.insert("status".to_string(), "critical".to_string());
 
 		let result = notifier.format_message(&variables);
-		assert_eq!(result, "*Alert*\n\nValue is 100 and status is critical");
+		assert_eq!(result, "Value is 100 and status is critical");
```

Comment:
I don't think the new implementation should be breaking tests. If it is, you should fix the implementation, not modify the tests :)